overrides/kubejs/server_scripts/earlygame.js

```diff
@@ -0,0 +1,110 @@
+
+
+onEvent('recipes', event => {
+
+    //Functions
+    function botaniaManaInfusion(output, input, mana, catalyst) {
+        event.custom({
+            type: 'botania:mana_infusion',
+            input: item.of(input),
+            output: item.of(output),
+            mana: mana,
+            catalyst: catalyst
+        }).id('kjs_' + output)
+    }
+
+    function elvenTrade(name, outputs, inputs) {
+        event.custom({
+            type: 'botania:elven_trade',
+            ingredients: [
+                ingredient.of(inputs)
+            ],
+            output: [
+                ingredient.of(outputs)
```

Comment:
This should probably be item.of instead, as ingredients support tags which I'm pretty sure cant be outputs.

Reply:
Ah. well I just woke up but the PR was already merged. I'll make Alego aware so that he can change it 👍